components/admin_console/system_users/system_users_dropdown/system_users_dropdown.tsx

```diff
@@ -200,6 +200,7 @@ export default class SystemUsersDropdown extends React.PureComponent<Props, Stat
                                 <li>
                                     <FormattedMessage
                                         id='deactivate_member_modal.desc.for_users_with_bot_accounts1'
+                                        // eslint-disable-next-line formatjs/no-multiple-whitespaces
                                         defaultMessage='This action deactivates {username}<p>'
```

Comment:
```suggestion
                                        defaultMessage='This action deactivates <p>{username}</p>'
```
should this be?

Reply:
Actually, it shouldn't even have a `p` tag. Looks like it was added by mistake in https://github.com/mattermost/mattermost-webapp/pull/10793
```suggestion
                                        defaultMessage='This action deactivates {username}'
```

Reply:
oh, yes removed.